docs/multicluster/user-guide.md

```diff
@@ -559,6 +559,42 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
+## Antrea Multicluster Pod to Pod Connectivity
+Since Antrea v1.9.0, Multi-cluster allows traffic between Pods in different clusters. If you want to enable this feature,
+the Pod CIDRs must be set in the ConfigMap `antrea-mc-controller-config` like below. You can edit the config in
+[antrea-multicluster-member.yml](../../multicluster/build/yamls/antrea-multicluster-member.yml) or using `kubectl edit`.
+
+```yaml
+apiVersion: v1
+data:
+  controller_manager_config.yaml: |
+    apiVersion: multicluster.crd.antrea.io/v1alpha1
+    kind: MultiClusterConfig
+    health:
+      healthProbeBindAddress: :8080
+    metrics:
+      bindAddress: "0"
+    webhook:
+      port: 9443
+    leaderElection:
+      leaderElect: false
+    serviceCIDR: ""
+    podCIDRs:
+      - "10.10.1.1/16"
+    gatewayIPPrecedence: "private"
+    endpointIPType: "ClusterIP"
+kind: ConfigMap
+metadata:
+  labels:
+    app: antrea
+  name: antrea-mc-controller-config
+  namespace: kube-system
+```
+
+The `podCIDRs` are the CIDRs routed by OVS. Normally, the `podCIDRs` is the cluster CIDRs. If it's left empty,
```

Comment:
Probably remove "The `podCIDRs` are the CIDRs routed by OVS."

Reply:
> Normally, the `podCIDRs` is the cluster CIDRs

This sentence is not clear. Maybe you mean "Normally, `podCIDRs` should be the value of `kube-controller-manager`'s `cluster-cidr` option, or the `kubeadmin` `pod-network-cidr` option when the cluster is created with `kubeadm`.